.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[target.'cfg(target_os = "windows")']
+rustflags = ["-Clink-args=/guard:cf /DYNAMICBASE /CETCOMPAT"]
+
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-Ctarget-feature=+crt-static"]

```

Comment:
@swernli why are static link crt on windows?

Reply:
Is the question why do it on Windows at all or why do it only on Windows (as in, we should do it for all platforms)? I think not depending on the client machine having the C runtime installed is good, but I only thought about and saw references to doing this on Windows. Do you thing we should do it more broadly or not do it all?

Reply:
IIRC, `+crt-static` is a no-op on platforms that don't support static linking, such that it _should_ be safe to include it on all targets.

Reply:
After some discussion, we landed at simplifying these flags here.